docs/product-assembly_pam-on-openshift/src/main/asciidoc/apb-deploy-proc.adoc

```diff
@@ -0,0 +1,35 @@
+[id='apb-deploy-proc']
+= Deploying a {PRODUCT} environment using the Automation Broker
+
+To deploy a {PRODUCT} environment using the Automation Broker, you must find the Ansible Playbook in the OpenShift catalog, run it, and set the parameters as required. 
+
+.Procedure
+
+. In the OpenShift Web UI, select *Add to Project -> Browse Catalog*.
+
+. In the search field, type {PRODUCT}.
+
+. Select the _{PRODUCT} {PRODUCT_VERSION} APB_ catalog entry.
+
+. Click the *Next* button.
```

Comment:
Here, next step, and anywhere else where "button" appears, revise. Just say "Click **Next**." Per IBM/RH style guides. Only specify "button" when it's unclear or doesn't have a name, like "Click the edit icon/button", where it's just some symbol or something.